styleguide/source/assets/scss/01-atoms/_checkbox.scss

```diff
@@ -26,7 +26,7 @@
     .ui-icon-background {
       box-shadow: none;
       border-radius: 0;
-      border: 1px solid $gray-50;
+      border: 1px solid $gray-7;
```

Comment:
did you intend to change the color here?

Reply:
@designerneil Yes, per our discussion the checkboxes should all reflect the generic forms Zeplin mockup. This change updates the border to reflect that.